webui/react/src/shared/styles/storybook.scss

```diff
@@ -8,6 +8,7 @@
  */
 
 #root {
+  background-color: var(--theme-stage);
```

Comment:
added this background color. Another option is `--theme-background` but it was too dark for dark mode, and this color isn't commonly visible in main screen of determined.
Any thoughts?